app/controllers/users_controller.rb

```diff
@@ -23,6 +24,8 @@ def create
 
   def show
     @user = User.find(params[:id])
+    @cookbooks = Cookbook.where(user_id: session[:user_id])
```

Comment:
anonymous users will need to be able to view the show page, but not the user's cookbooks.